src/components/LayoutTree.tsx

```diff
@@ -739,149 +743,167 @@ const LayoutTree = () => {
-                            // last={!nextChildId}
-                          />
-                        </div>
-                      )
+        <TransitionGroup>
+          {treeThoughtsPositioned.map(
+            (
+              {
+                belowCursor,
+                cliff,
+                depth,
+                env,
+                height,
+                indexChild,
+                indexDescendant,
+                isCursor,
+                isTableCol1,
+                isTableCol2,
+                key,
+                leaf,
+                path,
+                prevChild,
+                showContexts,
+                isLastVisible,
+                simplePath,
+                singleLineHeightWithCliff,
+                style,
+                thoughtId,
+                value,
+                width,
+                autofocus,
+                x,
+                y,
+              },
+              index,
+            ) => {
+              // List Virtualization
+              // Do not render thoughts that are below the viewport.
+              // Exception: The cursor thought and its previous siblings may temporarily be out of the viewport, such as if when New Subthought is activated on a long context. In this case, the new thought will be created below the viewport and needs to be rendered in order for scrollCursorIntoView to be activated.
+              // Render virtualized thoughts with their estimated height so that document height is relatively stable.
+              // Perform this check here instead of in virtualThoughtsPositioned since it changes with the scroll position (though currently `sizes` will change as new thoughts are rendered, causing virtualThoughtsPositioned to re-render anyway).
+              if (belowCursor && !isCursor && y > viewportBottom + height) return null
+
+              // Create a ref for each node. This helps to avoid the `findDOMNode is deprecated` warning.
+              const nodeRef = createRef<HTMLDivElement>()
+
+              const nextThought = isTableCol1 ? treeThoughtsPositioned[index + 1] : null
+              const previousThought = isTableCol1 ? treeThoughtsPositioned[index - 1] : null
+
+              // Adjust col1 width to remove dead zones between col1 and col2, increase the width by the difference between col1 and col2 minus bullet width
+              const xCol2 = isTableCol1 ? nextThought?.x || previousThought?.x || 0 : 0
+              // Increasing margin-right of thought for filling gaps and moving the thought to the left by adding negative margin from right.
+              const marginRight = isTableCol1 ? xCol2 - (width || 0) - x - (bulletWidth || 0) : 0
+
+              // Speed up the tree-node's transition by 50% on New (Sub)Thought only.
+              const isNewThought = value === ''
+              const layoutDuration = isNewThought ? `layoutNodeAnimationFastDuration` : `layoutNodeAnimationDuration`
```

Comment:
This duration does not appear to have the intended effect (or any effect?). Since the duration is so short, I updated `layoutNodeAnimationDuration` to `5000` ms in the durations config to test it locally. Doing so doesn't seem to have lengthened the duration of any animations for a new subthought:

https://github.com/user-attachments/assets/1f82445d-fc0f-48ba-a76e-56955c74221b

I also increased the value of `layoutNodeAnimationDuration` with the same unchanged results.